docs/deployment.rst

```diff
@@ -4,21 +4,37 @@ Deployment
 
 .. contents::
 
-Environments
-============
+Environments and deploys
+========================
 
-Mozilla Symbol Server is deployed in 3 different environments:
+Buildhub2 has two server environments: stage and prod.
 
-1. Production (To Be Decided)
 
-2. Stage (To Be Decided)
+Stage
+-----
 
-The stage environments is updated on every new tag created.
+Stage is at: https://stage.buildhub2.nonprod.cloudops.mozgcp.net/
 
-The prod environment requires a tag and a manual task by CloudOps.
+To deploy to stage, tag the master branch and push the tag::
+
+   $ make tag
+
+
+.. Note::
+
+   This assumes the remote is named "origin".
```

Comment:
That's soooo peterbe!

Reply:
Who would name it anything else?

Reply:
Some people call the mozilla repo "upstream" and the myusername repo the "origin". Or something like that. 

I personally try to be consistent and always call the true source of truth repo "origin" and if/when I have a fork, I call that "peterbe". Makes it easier on me for my [git scripts](https://github.com/peterbe/gg).

Reply:
I call it "official". It's not a big deal--this is mostly a note for me to remember. Plus I might at some point tweak the script.

Reply:
This has surprised me a bit in the past, everyone seems to have a different workflow for remote naming which is natural for them. For scripts, since there is no harm in having a remote listed twice, I've been considering verbose remote names, like:

```Makefile
set-upstream-remote:
	git remote remove mozilla-official-repo
	git remote add mozilla-official-repo git@github.com:mozilla-services/buildhub2.git
	git fetch mozilla-official-repo

tag: set-upstream-remote
	@bin/make-tag.py
```
 
I haven't implemented this in any project yet, so it needs some debugging.

Reply:
Oh, interesting. I was thinking we could have it figure out the remote. Something like this:

```
REMOTE=$(git remote -v | grep mozilla-services/buildhub2 | awk '{print $1}' | head -n 1)
```

I don't know if that works right or not. It might have carriage returns in it or something. Also, it might not work in all cases. We might just have the user set is as an env var.